_pages/en/it-picture-medium-term.md

```diff
@@ -132,11 +146,20 @@ En conclusion, ...
 - https://cloud.google.com/solutions/devops/devops-measurement-wip-limits
 - https://cloud.google.com/solutions/devops/devops-measurement-visual-management
 
+Business lines understand their accountability in sustaining the IT systems used to enable their business.
+This means part of their budget is set to assure technical debt is measured, exposed to the business line, and handled in a timely manner
+
+Operations have moved towards a Site Reliability Engineering (SRE) practice where operations is treated like a software practice.
+ESDC applications are managed, maintained, and monitored using software.
+SREs collect data throughout the IT value chain and aggregate it to different audiences.
+
 Teams have established Work in progress (WIP) limits based on team capacity and workload assigned, in order to prioritize and focus on completing a small number of high-priority tasks. Once the process is implemented accurately, this will result in faster lead times, higher quality, lower costs, and less waste.
 
-The IT branch is collecting data from key areas and appropriate metrics throughout the value chain and ensuring that the right data is being collected.The collected data is available, shared, and used to make decisions across ESDC (not just IT).
-Business lines performance metrics are used as benchmark and the accountability of IT's impact to ESDC's mandate is shared.
-Teams setup proactive notification that generate alert notifications when monitored values approach known failure thresholds.
+SRE provides proactive failure notification, using key indicators and thresholds in their monitoring software, and get alerted before an application fails to pro-actively resolve issues before they impact users.
 Following the diagnoses of incidents, indicators that could have predicted the incident are added to monitoring.
 
-En conclusion, ...
+Most of the Application Portfolio Management (APM) practice measures the health of applications using metrics directly from the SRE teams`s software, not using out dated documentations.
```

Comment:
Not mention APM?

Reply:
You mean to remove it?
We need to keep APM (for the medium term at least). It's also a MAF requirements to provide the GC with the health of aging infrastructure. But it's also a tool ESDC can use for investment decisions. The Technical Debt team (APM used to be within EA but now has moved to the Technical Debt team under JL). Right now though, APM is very manual, is filled by the wrong personnel (should be higher up), and is not linked to the business lines' services, which does not provide the "health" of those business services.

Removed the mention of SRE though